py/client/pydeephaven/session.py

```diff
@@ -514,14 +514,16 @@ def query(self, table: Table) -> Query:
         return Query(self, table)
 
     def input_table(self, schema: pa.Schema = None, init_table: Table = None,
-                    key_cols: Union[str, List[str]] = None) -> InputTable:
-        """Creates an InputTable from either Arrow schema or initial table. When key columns are
-        provided, the InputTable will be keyed, otherwise it will be append-only.
+                    key_cols: Union[str, List[str]] = None, blink_table: bool = False) -> InputTable:
+        """Creates an InputTable from either Arrow schema or initial table.  When blink_table is True, the InputTable
+        will be a blink table. When blink_table is False (default), if key columns are provided, the InputTable will be
+        keyed, otherwise it will be append-only.
```

Comment:
```suggestion
        """Creates an InputTable from either Arrow schema or initial table.  When blink_table is True, the InputTable
        will be a blink table. When blink_table is False (default), the InputTable will be
        keyed if key columns are provided, otherwise it will be append-only.
```